GliaWidgets/Sources/ViewModel/Call/CallViewModel+CameraFlip.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+extension CallViewModel {
+    func showCameraFlipIfNeeded() {
+        let flipCameraCallback: Cmd?
+
+        defer {
+            self.action?(.setCameraFlip(flipCameraCallback))
+        }
+
+        do {
+            let cameraDeviceManager = try environment.cameraDeviceManager()
+            let devices = cameraDeviceManager.cameraDevices().filter { $0.facing == .front || $0.facing == .back }
+            // Camera flip only makes sense when there are at least two camera devices.
+            guard devices.count >= 2 else {
+                flipCameraCallback = nil
+                return
+            }
```

Comment:
I'd add array of facing cases and rely on its count. Does it make sense for you?
```suggestion
            let cameraDeviceManager = try environment.cameraDeviceManager()
            let cameraDeviceFacingTypes: [PutCorrectFacingType] = [.front, .back]
            let devices = cameraDeviceManager.cameraDevices().filter { cameraDeviceFacingTypes.contains($0) }
            // Camera flip only makes sense when there are at least two camera devices.
            guard devices.count >= cameraDeviceFacingTypes.count else {
                flipCameraCallback = nil
                return
            }
```

Reply:
On one hand it makes sense, to limiting camera devices to have both `front` and back `facing` to allow true meaning of flip. On the other hand we would discard changing cameras if there are two `back` cameras while `front` one is unavailable and vice versa. I would prefer for now to allow changing of camera even if there are only two or more same facing devices. But it would be good to hear from others about it. cc: @AndriiHorishniiMOC @yurii-glia @gersonnoboa @rasmustautsglia

Reply:
I also like the approach of going through the cameras instead even though the meaning of "flipping" gets thrown away by this implementation. But right now, if Apple decides to switch between two cameras at the back for some reason, we would be able to support it without changing the SDK, which is a nice thing to have.

Reply:
Agree with @igorkravchenko  & @gersonnoboa